recipes/django-treebeard/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "django-treebeard" %}
+{% set version = "4.1.2" %}
+{% set sha256 = "a11bb3dfc60004adda12a4e2e2fd1f691b36d7aa302604542926d5d2209687fd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - django >=1.8
+    - pytest
```

Comment:
Is `pytest` actually required to run this package? It looks like a testing-only req:

https://github.com/django-treebeard/django-treebeard/blob/master/setup.py#L30

Reply:
@synapticarbors : Thanks for your feedback. I have removed pytest from run requirements.